src/components/VmDisks/index.js

```diff
@@ -26,7 +26,14 @@ class VmDisks extends Component {
 
     let classes = style['disks-list']
 
-    let disksToRender = disks.sort((a, b) => a.get('name').localeCompare(b.get('name')) - (a.get('bootable') ? 1000 : 0))
+    let disksToRender = disks.sort((a, b) => {
+      const aBoot = a.get('bootable')
+      const bBoot = b.get('bootable')
+
+      return aBoot && !bBoot ? -1
+        : !aBoot && bBoot ? 1
+        : a.get('name').localeCompare(b.get('name'))
```

Comment:
Can we please use number aware sorting:

```js
const b = ['r', 'ř10', 'ř9', 'ř']
b.sort((a, b) => a.localeCompare(b, undefined, { numeric: true }))
// b is Array [ "r", "ř", "ř9", "ř10" ]
```

Reply:
Excellent point.  I've added that and some tests so it's a bit more obvious what is intended.